### PR TITLE
A Better README file

### PR DESCRIPTION
**Pull request by ixahmedxi**
I create a new documentation file because the original one had some problems, I added the following:

* A command to install the package with yarn
* A license section at the end
* The package name as a header at the start
* Fixed the wrong syntax highlighting, using html as the language is better than javascript since markdown has no JSX support.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,48 +25,74 @@ import FlexContainer from 'react-styled-flexbox';
...
+* justifySpaceAround
+* justifyFlexEnd
+* justifyCenter
+
+If you are not familiar with the flexible box model please review the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Using_CSS_flexible_boxes)
+
+## License
+MIT License
+
+Copyright (c) 2017 Steven Akins
```
Comment (snikas) [inline README.md]: Please change this to Copyright (c) 2017 Raycom Media, Inc.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,22 @@
-[![Build Status](https://travis-ci.org/snikas/react-styled-flexbox.png?branch=master)](https://travis-ci.org/snikas/react-styled-flexbox)
```
Comment (snikas) [inline README.md]: I'd still like to show this badge. Did you mean to remove it?

**Review by snikas — CHANGES_REQUESTED** [review]
Thanks for the README update!
Please update the copyright line and I'll approve.

**Comment by ixahmedxi** [comment]
I changed the file now.

**Pull request merged**